website/src/pages/index.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import clsx from 'clsx';
+import Heading from '@theme/Heading';
+import Translate, {translate} from '@docusaurus/Translate';
+import styles from './styles.module.css';
+
+function HeroBanner() {
+  return (
+    <div className={styles.hero} data-theme="dark">
+      <div className={styles.heroInner}>
+        <Heading as="h1" className={styles.heroProjectTagline}>
+          <span
+            className={styles.heroTitleTextHtml}
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={{
+              __html: translate({
+                id: 'homepage.hero.title',
+                message:
+                  'A customizable cloud native policy controller that helps <b>enforce policies and strengthen governance</b>',
+                description:
+                  'Home page hero title, can contain simple html tags',
+              }),
+            }}
+          />
+        </Heading>
+        <div className={styles.indexCtas}>
+          <Link className="button button--primary" to="/docs">
+            <Translate>Get Started</Translate>
+          </Link>
+          <Link className="button button--info" to="https://docusaurus.new">
+            <Translate>Browse the Policy Library</Translate>
+          </Link>
+          <span className={styles.indexCtasGitHubButtonWrapper}>
+            <iframe
+              className={styles.indexCtasGitHubButton}
+              src="https://ghbtns.com/github-btn.html?user=open-policy-agent&amp;repo=gatekeeper&amp;type=star&amp;count=true&amp;size=large"
+              width={160}
+              height={30}
+              title="GitHub Stars"
+            />
+          </span>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function ContributingCompanies() {
+  return (
+    <section className={styles.companies}>
+      <div className="container">
+        <div className="row">
+          <div className={clsx('col text--center')}>
+            <h2>Contributed by the community in collaboration with</h2>
+          </div>
+        </div>
+        <div className="row">
+          <div className={clsx('col col--3 text--center')}>
+            <a href="https://cloud.google.com/">
+              <img className={styles.companySvg} src="img/google_cloud_logo.svg" />
```

Comment:
Google cloud's logo has changed to Google [in colored letters] cloud. I think the resources can be viewed here?

https://standards.google/guidelines/cloudstyle/logo/

Reply:
Looks like that might be an internal site. I did see the lettered logo on the branding page, but it's a bummer it looks like they don't have a vector graphic there :(

Reply:
Argh. Does this download work? https://kstatic.googleusercontent.com/files/51647deaf002c48e035eabd032ef9aa7f637bd5a5cecd3e26932fc10df23e6a1a6fcfea7410e54b25dea7da59245f41ecb181d5920bf963f8f217994f71bf1d2

Reply:
If not, I can email it. It has some svgs